src/Kunstmaan/AdminBundle/Resources/ui/scss/default-theme/general/_login.scss

```diff
@@ -14,8 +14,12 @@
 /* Title
    ========================================================================== */
 .login__title {
-    margin: 20vh auto 2rem;
+    margin: 10vh auto 2rem;
```

Comment:
I think it's best to use a vh fallback mixin here. iOS7 has poor vh unit support